myapp/getClientInfo.js

```diff
@@ -0,0 +1,16 @@
+const request = require('request');
+const app = require('./app.js');
+
+getClientInfo = function () { 
+  app.get('/clients/:clientid', function (req, res) {
+    request('https://api.createsend.com/api/v3.1/clients/'+req.params.clientid+'.json', function (err, response, data) {
+      if(err) {
+        res.send(err);
```

Comment:
if you don't break via `return` from here, the code will still continue to get exectued and would reach the lines below!